design-at-cornell/src/areas-of-study/dashboard/ApplyTags.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { 
+  ApplyTagsContainer
+} from '../AreasOfStudyStyles';
+
+const ApplyTags = () => {
+  const designAreaTags = ['Art + Design', 'Digital Design', 'Industrial Design', 'Policy Design', 'Spatial Design']
+  const schools = ['AAP', 'Agriculture and Life Sciences', 'Arts and Sciences', 'Engineering', 'Human Ecology', 'ILR', 'SC Johnson School of Buisness']
+
+  const checkboxes = (tags: string[]) => (
+    <form>
+      {tags.map(tag => (
+        <div>
+          <input key={tag} type="checkbox" id={tag} />
+          <label htmlFor={tag}>{tag}</label>
+        </div>
+      ))}
+    </form>
+ )
```

Comment:
Is it possible to align the text and checkboxes a bit more? It's a bit shifted down, but also no biggie if we can't.
<img width="143" alt="Screen Shot 2021-04-01 at 11 34 48 PM" src="https://user-images.githubusercontent.com/15709162/113377632-b5532780-9329-11eb-966e-02e9dab15321.png">

Reply:
I added vertical alignment to the checkboxes, so they should theoretically be aligned with the text. But they still don't seem to be 100% aligned (though much better than before) and I don't know what to fix that :(